tests/evalhub/test_adapter.py

```
@@ -165,3 +170,17 @@ def test_adapter_handles_runner_failure():
+
+def test_adapter_rejects_cursor_runner():
+    with tempfile.TemporaryDirectory() as tmpdir:
+        tmpdir = Path(tmpdir)
+        eval_yaml = _make_eval_yaml(tmpdir, runner_type="cursor")
+        config = _StubJobSpec()
+        callbacks = _StubJobCallbacks()
+
+        with patch("agent_eval.evalhub.adapter._framework_adapter_init"):
+            from agent_eval.evalhub.adapter import AgentEvalAdapter
+            adapter = AgentEvalAdapter(eval_config_path=str(eval_yaml))
+            with pytest.raises(ValueError, match="does not support runner.type: cursor"):
```

Ruff (0.16.2)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)